### DISCUSSION TITLE
Mh/s instead of GH

### DISCUSSION THREAD

**Discussion by seblezinzin** [onemorebsmith/kaspa-stratum-bridge#62] [General]
Hello

Why does graphana show MH/s instead of GH ?

thanks 

![grap](https://user-images.githubusercontent.com/13225357/203085838-95c4f136-cdbf-494d-b59c-8652619d5d8e.png)

**Comment by jvxis** [comment]
You need to edit the panel and change it to kilohash.
![image](https://user-images.githubusercontent.com/108929149/203137116-0f866c06-fe19-49c2-9049-e688663bd8ad.png)

**Reply by seblezinzin** [reply]
thanks a lot.

Other question how to get block mine by worker ?

**Reply by jvxis** [reply]
Hi, you can use this query COUNT BY(wallet,worker) (rate(ks_mined_blocks_gauge{wallet=~"${wallet_filter}"}[1y]))
